install/consul/istio.yaml

```diff
@@ -1,3 +1,5 @@
+# GENERATED FILE. Use with Docker-Compose and Consul
+# TO UPDATE, modify files in install/consul/templates and run install/updateVersion.sh
```

Comment:
would you also put consul's istio.yaml in this install/consul/templates and rename it to istio.yaml.tmpl?  (I'm not seeing the change for that, thus the ask.)

Reply:
i saw the missing file you just added, so this one is good :)